tests/phpunit/tests/term.php

```diff
@@ -308,4 +308,87 @@ public function test_orphan_category() {
 		$cat_id2 = self::factory()->category->create( array( 'parent' => $cat_id1 ) );
 		$this->assertWPError( $cat_id2 );
 	}
+
+	/**
+	 * @ticket 58087
+	 * @coversNothing
+	 *
+	 * @param stdClass $actual An instance of stdClass containing the values to initialize the WP_Term object.
+	 * @param array $expected An array containing the expected values for the WP_Term object.
+	 *
+	 * @dataProvider data_get_raw_data_should_return_correct_values
```

Comment:
```suggestion
	 * @dataProvider data_get_raw_data_should_return_correct_values
	 *
	 * @param stdClass $actual An instance of stdClass containing the values to initialize the WP_Term object.
	 * @param array $expected An array containing the expected values for the WP_Term object.
```

Reply:
Thank you for the review, @mukeshpanchal27.
I agree with the proposed changes; however, I don't think the `@covers` tag should be removed.
Fixed in 0e8df1a1a6d64170a8b97e193f49d48571e23039